lib/tasks/router_data.rake

```diff
@@ -1,24 +0,0 @@
-namespace :router_data do
```

Comment:
There's a bit more context in the PR itself, but I agree it looks like it can be removed now: https://github.com/alphagov/publisher/pull/540

...along with other methods that were only added to support this rake task:  https://github.com/alphagov/publisher/commit/25b3355105bbff6120ac9e62f104f30ed66ce907#diff-6405599d2d28cbb948fbe5d185635660R18-R28

Reply:
Sure, I've updated 👍